individual-reflection.md

```diff
@@ -9,48 +9,61 @@ Answer the following comprehension questions **within this file.** Write your an
 ### `GET` Request Review
 
 1. Describe a GET request that your project makes, and the high-level description of what it does
-    - Answer:
+    - Answer: the GET request sends the requests to the server reqesting to send data back to the client.
 1. What is the verb of this request?
-    - Answer:
+    - Answer: GET
 1. What is the path (or the URL, or endpoint) of this request?
-    - Answer:
+    - Answer: the path tells the server what resource that the client is interested in. For example, "https://slack.com/api/channels.list"
 1. What are the query params (the additional data sent with the request, besides the verb and the path)?
-    - Answer: 
+    - Answer: the query params are additional data that are sent with the request along with the path. For example, Token is an addtional required parameter that we need to pass in along with the url path. 
 1. What is the syntax used to make this request? (Copy and paste a code snippet here)
     - Answer:
       ```ruby
-      # Copy and paste your answer below this comment
+      def self.get(url)
+        response = HTTParty.get(url, query: {token: ENV['SLACK_TOKEN']})
+
+        if response.code != 200 || response["ok"] == false
+      raise SlackAPIError, "We encountered a problem: #{response["error"]}"
+        end
 
-      # Copy and paste your answer above this comment
+        return response
+        end
       ```
 1. What does the program do if the response comes back with a status code of 200?
-    - Answer: 
+    - Answer: OK/success
```

Comment:
What does your program (your Ruby code) do if the response has a status code of 200? What is the logic that your Ruby code follows after getting a response back?